_posts/2023-03-20-semantic-science-benchmarks.md

```diff
@@ -0,0 +1,318 @@
+
+b~i​=bi​/∣∣b∣∣
+
+Given a list of scores *b* for BM25 and *n* for neural search, we can combine them into the combination score *s* as follows:
+
+si​=⎩⎪⎪⎪⎪⎪⎪⎨⎪⎪⎪⎪⎪⎪⎧​2b~i​+n~i​​,b~i​n~i​​,b~i​+n~i​2b~i​n~i​​,​arithmetic meangeometric meanharmonic mean​
+
+The fine-tuned models have been trained for 10 epochs on the synthetic queries created by the query generator. For smaller datasets, such as nfcorpus, arguana and fiqa, we created 32 queries per passage, while for larger datasets we created fewer queries per passage. In particular, we created 26 queries per passage for cqadupstack and 16 for Amazon ESCI.
+
+The following table contains the results for combining these scores on the 10 test datasets. 
+
+
+|	|BM25	|TAS-B	|TAS-B <br> with <br>L2 norm <br>(arithmetic mean)	|TAS-B <br>with <br>L2 norm <br>(harmonic mean)	|TAS-B <br>with <br>L2 norm <br>(geometric mean)	|Custom	|Custom <br>with <br>L2 norm <br>(arithmetic mean)	|Custom <br>with <br>L2 norm <br>(harmonic mean)	|Custom<br> with <br>L2 norm <br>(geometric mean)	|
+|---	|---	|---	|---	|---	|---	|---	|---	|---	|---	|
+|nfcorpus	|0.34281	|0.31886	|0.34607	|0.35046	|0.34845	|0.3014	|0.36919	|0.36458	|0.36727	|
+|trec-covid	|0.68803	|0.48115	|0.73248	|0.73094	|0.73533	|0.57726	|0.75211	|0.78825	|0.79013	|
+|arguana	|0.47163	|0.42704	|0.48523	|0.48167	|0.4838	|0.492	|0.52722	|0.51076	|0.52572	|
+|fiqa	|0.25364	|0.30024	|0.28911	|0.2812	|0.28243	|0.31413	|0.36422	|0.32627	|0.34961	|
+|scifact	|0.69064	|0.64276	|0.68598	|0.69132	|0.68722	|0.62275	|0.72845	|0.72227	|0.72749	|
+|dbpedia	|0.32016	|0.38423	|0.34081	|0.39482	|0.3586	|0.34194	|0.37337	|0.39159	|0.39235	|
+|quora	|0.80771	|0.83516	|0.83587	|0.84714	|0.84105	|0.85524	|0.87396	|0.86943	|0.8719	|
+|scidocs	|0.16468	|0.14859	|0.16956	|0.16945	|0.16969	|0.15415	|0.18382	|0.1805	|0.18351	|
+|cqadupstack	|0.3253	|0.3144	|0.3434	|0.3374	|0.34	|0.3566	|0.3673	|0.3518	|0.3766	|
+|Amazon ESCI	|0.08111	|0.07061	|0.08525	|0.08761	|0.08662	|0.07418	|0.09082	|0.09033	|0.09079	|
+|Average performance against BM25 (in %)	|n/a	|-3.77257	|4.71726	|6.43925	|5.24445	|-0.24703	|13.91945	|12.15005	|14.62501	|
+
+
+
+We found that harmonic combination works best for the pretrained TAS-B model, while arithmetic combination works better for the fine-tuned custom model.  Note that for a given query there could be documents that are only present in the dense results and not the BM25 results. In such cases, we assume the BM25 score for those documents is zero. Conversely, if there are documents that are only present in the BM25 results, we assume the neural query score for those documents is zero. 
```

Comment:
Important: arithmetic changes to  "while arithmetic and geometric combination work best"